server/handler.go

```diff
@@ -130,7 +141,134 @@ func (h *Handler) ComQuery(
 	c *mysql.Conn,
 	query string,
 	callback func(*sqltypes.Result) error,
-) (err error) {
+) error {
+	return h.doQuery(c, query, nil, callback)
+}
+
+func bindingsToSql(bindings map[string]*query.BindVariable) (map[string]sql.Expression, error) {
+	res := make(map[string]sql.Expression, len(bindings))
+	for k, v := range bindings {
+		v, err := sqltypes.NewValue(v.Type, v.Value)
+		if err != nil {
+			return nil, err
+		}
+		switch {
```

Comment:
Seems like these would be better expressed as a switch on v.Type(), no?

Reply:
Yeah, maybe...I was using a few expressions like `sqltypes.IsUnsigned(v.Type())`, so I went with this approach...could change it or split it into two groups...